### PR TITLE
Version bump and changelog for 0.5.0

### PR DESCRIPTION
**Pull request by ptomato**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
+  - Temporal.Instant.toDateTime
+- Conversion methods that add information, such as
+  Temporal.PlainYearMonth.toPlainDate(), now always take a property bag
+  with the fields to add.
+- Temporal.Instant.toString() now takes one argument, an options bag.
+  The time zone may be specified as the value of the `timeZone` property
+  in this argument.
+- The toString() methods of Temporal.PlainDate, Temporal.PlainDateTime,
+  Temporal.PlainMonthDay, Temporal.PlainTime, and
+  Temporal.PlainYearMonth now take a `calendar` property which controls
```
Comment (justingrant) [inline CHANGELOG.md:78]: You recently renamed the time zone equivalent of this property to `timeZoneName`, correct?  Seems like this should be renamed to `calendarName` for consistency. I can PR this.

**Review by justingrant — APPROVED** [review]
Looks good. I had one question inline.

File: CHANGELOG.md (hunk 1 above)
Comment (justingrant) [reply CHANGELOG.md:78]: PR is ready if it'd be welcome

Comment (ptomato) [reply CHANGELOG.md:78]: Sure, go for it.

**Review by ryzokuken — APPROVED** [review]
LGTM, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (justingrant) [reply CHANGELOG.md:78]: Done. #1138

**Pull request merged**